core_backend/app/contents/models.py

```diff
@@ -229,14 +273,19 @@ async def update_content_in_db(
 
 
 async def increment_query_count(
```

Comment:
We should do the updating the content history table in a separate function (esp if we move it to the AQ module) or change this function name and docstring to reflect its actions.